README.md

```diff
@@ -1,92 +1,198 @@
+  allowed_return_urls                          secret           architectio/kratos       local        create  complete  0s                                                  
+  kratos                                       database         architectio/kratos       local        create  complete  2s                                                  
+  architectio/kratos/deployment/kratos/kratos  databaseUser     architectio/kratos       local        create  complete  1s                                                  
+  kratos                                       deployment       architectio/kratos       local        create  complete  1s                                                  
+  ui                                           deployment       architectio/kratos       local        create  complete  0s                                                  
+  kratos-public                                service          architectio/kratos       local        create  complete  0s                                                  
+  kratos-admin                                 service          architectio/kratos       local        create  complete  0s                                                  
+  frontend                                     service          architectio/kratos       local        create  complete  0s                                                  
+  kratos-public                                ingressRule      architectio/kratos       local        create  complete  0s    http://kratos-public.local.127.0.0.1.nip.io/  
+  auth                                         ingressRule      architectio/kratos       local        create  complete  0s    http://auth.local.127.0.0.1.nip.io/
+```
+
+#### Bonus: Dev environments
+
+Want to quickly test your components and application code? We've got you covered! Just run `arcctl up .` in the directory your component lives in andspecify a datacenter to power your environment. Architect will automatically create an environment, deploy the component to it in debug mode, and then stream the application logs to your terminal. Yes, this is just like docker-compose!
+
+```sh
+$ arcctl up . --datacenter local
+```
+
+## For DevOps
+
+### Key features
+
+### Getting started
+
+#### 1. Create a few datacenters
+
+Every datacenter you create becomes someone that you or your team can create and run cloud environments on top of. We've created a few sample datacenters for you to try, but they all use our [datacenters framework](./src/datacenters/) which means you're able to tweak them or create entirely new ones.
+
+```sh
+# A datacenter that runs components locally
+$ arcctl apply datacenter local https://raw.githubusercontent.com/architect-team/arcctl/main/examples/datacenters/local.yml
+
+# A datacenter that runs workloads on GCP Cloud Run
```

Comment:
Setting up GCP is complex, get why we are promoting this, but our getting started guide might want to link to the guide to setup the credentials here.